src/equations/laplace_diffusion_2d.jl

```diff
@@ -0,0 +1,46 @@
+@doc raw"""
+    LaplaceDiffusion2D{T} <: AbstractParabolicEquations{2, 1}
+
+`LaplaceDiffusion2D` represents a scalar diffusion term ``\nabla \cdot (\kappa\nabla u))``
+applied to each solution component.
+"""
+struct LaplaceDiffusion2D{T} <: AbstractLaplaceDiffusionEquations{2, 1}
+  diffusivity::T
+end
+
+# no orientation specified since the flux is vector-valued
+function flux(u, grad_u, equations::LaplaceDiffusion2D)
+  dudx, dudy = grad_u
+  return equations.diffusivity * dudx, equations.diffusivity * dudy
```

Comment:
It returns a tuple in this case, which should also be immutable and fast. We can change it to an SVector for consistency, but if we introduce orientations into the viscous flux it would also fix this issue.

Reply:
For consistency, I'd prefer to use `SVector`s, since at least in the `TreeMesh` codes, using tuples has caused issues in the past
```suggestion
  return SVector(equations.diffusivity * dudx, equations.diffusivity * dudy)
```

Reply:
Done